src/wizard/Step5/components/PortfolioSummary.vue

```diff
@@ -82,11 +82,13 @@ export default class PortfolioSummary extends mixins(ApplicationData) {
           "Ensures your team can access their workspaces within the cloud console",
         name: "addteammembers",
       },
-    ].filter((item) => this.$store.state.erroredSteps.indexOf(item.id) > -1);
+    ].filter(
+      (item) => this.$store.state.wizard.erroredSteps.indexOf(item.id) > -1
+    );
   }
 
   created(): void {
-    this.portfolio = this.$store.getters.getPortfolio;
+    this.portfolio = this.$store.getters["wizard/getPortfolio"];
     this.taskOrders = this.$store.getters.getTaskOrders;
```

Comment:
when `getTaskOrders` gets moved to wizard getters, don't forget about this guy

Reply:
yup..  Fixed this... I just referenced it from the task orders store...  Thanks